media/js/firefox/new.js

```diff
@@ -27,6 +62,21 @@
                 // (from a download button) then we want them to see the same scene 2
                 // as non-firefox users and initiate a download
                 $html.addClass('firefox-latest');
+
+                uiTourGetConfiguration('appinfo', function(config) {
```

Comment:
:sob: That is indeed accurate. It's been working so far, but we can't rely on UITour taking long enough for the DOM to initialize. Will update.